pkg/gofr/datasource/scylladb/internal.go

```diff
@@ -0,0 +1,273 @@
+package scylladb
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"strings"
+	"time"
+
+	"github.com/gocql/gocql"
+	"github.com/stoewer/go-strcase"
```

Comment:
It was only a suggestion

Let's wait for a maintainer feedback here

You are using this conversion for formatting the tag from columns right?

Maybe it could be done differently

Like using a lower case comparison in get fields

I'm unsure if Scylladb supports having columns with different cases

Reply:
This one was resolved without a comment

I'm unsure it was addressed

@Umang01-hash @aryanmehrotra 

What do you think about adding a dependency for this?